<launch>
    <arg name="odom_topic" value="/pos_vel_mocap/odom_TA" />
    <arg name="odom_cone_topic" value="/pos_vel_mocap/odom_cone" />
    <node pkg="uart_odom" type="uart_odom" name="uart_odom" output="screen">
        <remap from="~odom" to="$(arg odom_topic)" />
        <remap from="~odom_cone" to="$(arg odom_cone_topic)" />
        <rosparam command="load" file="$(find uart_odom)/config/uart_odom_config.yaml" />
        <param name="work_mode" value="real-time" />
        <param name="js_ctrl_mode" value="feedback" />
        <param name="serial_port" value="/dev/tty.uwb.master"/>
    </node>
    <node pkg="pos_vel_mocap" type="pos_vel_mocap" name="pos_vel_mocap" output="screen" required="true">
        <rosparam file="$(find pos_vel_mocap)/config/calib.yaml" command="load" />
    </node>
	<node pkg="mocap_optitrack" type="mocap_node" name="mocap_node" respawn="false" launch-prefix="" required="true" output="log">
        <rosparam file="$(find mocap_optitrack)/config/mocap.yaml" command="load" />
    </node>
</launch>
